src/frontend/index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Internet Identity</title>
-    <link rel="shortcut icon" href="/favicon.ico" />
+    <link rel="shortcut icon" href="/src/assets/favicon.ico" />
```

Comment:
So we can set the cache-control header of the favicon and add a filehash to it so it too is cached. It's another 15kb and in my tests it could take quite a bit of time to load on some refreshes.

Reply:
Noted.